pkg/validate/image.go

```diff
@@ -221,7 +220,7 @@ func pullImageList(c internalapi.ImageManagerService, imageList []string, podCon
 	for _, imageName := range imageList {
 		ids = append(ids, framework.PullPublicImage(c, imageName, podConfig))
 	}
-	return ids
+	return slices.Compact(ids)
```

Comment:
should we sort before compact? It looks like docs are saying it is deduping consequetive values, not all unique values

Reply:
That's tricky, the tests expect the ID's in the same order as the images, but I guess we can sort the image list and dedup the IDs then.

Reply:
Yeah, that depends on the test. I'll fix that up.